spec/support/test_contamination.rb

```diff
@@ -0,0 +1,19 @@
+module Spec
+  module Support
+    module TestContamination
+      def self.setup
+        TOPLEVEL_BINDING.eval('self').method(:include).owner.prepend(patched_include)
```

Comment:
I guess this is nicer than a typical monkey patch in that it's a method call and not just reopening Object on load.  :+1: Nice.